src/react/components/StatisticTable.js

```diff
@@ -71,12 +77,27 @@ const StatisticTable = () => {
   const [competition, setCompetition] = useState({});
   const [rankings, setRankings] = useState([]);
 
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setRankings(
+        rankings.map(ranking => {
+          return { ...ranking, visible: !ranking.visible };
+        })
+      );
+    }, 7000);
```

Comment:
please use a constant variable for each "number"

Reply:
f.ex const TIME_BETWEEN_SWITCHING - weißt schon

Reply:
Haha, sehr gut 😁

Reply:
nur so am rande, _for example_ wird mit "e.g." abgekürzt :)